seatunnel-api/src/main/java/org/apache/seatunnel/api/sink/SeaTunnelSink.java

```diff
@@ -135,4 +136,13 @@ default Optional<Serializer<CommitInfoT>> getCommitInfoSerializer() {
     default Optional<Serializer<AggregatedCommitInfoT>> getAggregatedCommitInfoSerializer() {
         return Optional.empty();
     }
+
+    /**
+     * Get the catalog table of the sink.
+     *
+     * @return Optional of catalog table.
+     */
+    default Optional<CatalogTable> getWriteCatalogTable() {
```

Comment:
cc @hailin0

Reply:
Please add test case in https://github.com/apache/seatunnel/blob/dev/seatunnel-dist/src/test/java/org/apache/seatunnel/api/connector/ConnectorSpecificationCheckTest.java#L50 to make sure all connector sink implement `getWriteCatalogTable`

Reply:
> Please add test case in https://github.com/apache/seatunnel/blob/dev/seatunnel-dist/src/test/java/org/apache/seatunnel/api/connector/ConnectorSpecificationCheckTest.java#L50 to make sure all connector sink implement `getWriteCatalogTable`

Done.